libraries/ESP8266WiFi/src/WiFiUdp.cpp

```diff
@@ -118,7 +118,7 @@ int WiFiUDP::available() {
     if (!result) {
         // yielding here will not make more data "available",
         // but it will prevent the system from going into WDT reset
-        optimistic_yield(1000);
+        optimistic_yield(10000);
```

Comment:
Same comment as above.  I think the existing //comments are somewhat misleading...the return from this call to `available` won't change but the context might actually have gotten some data due to letting the wifi/etc. stack run.

I'd make the delay consistent with the TCP one, though (100).